addons/mail/static/src/new/composer/composer.js

```diff
@@ -286,7 +286,20 @@ export class Composer extends Component {
             target: "new",
             context: context,
         };
-        await this.env.services.action.doAction(action);
+        const options = {
+            onClose: () => {
+                this.reset();
```

Comment:
`reset` => `clear`

`reset` gives the impression it's a refresh, whereas `clear` is clearer (no pun intended) on stating it clears the content.

I also took the opportunity of this PR to rename `AttachmentUploader/reset` to `clear`.